### ISSUE TITLE
doesn't work with RC6

### ISSUE THREAD

**Issue by k1ng440** [ngrx/store-log-monitor#10]
hello i am getting follow error:
Error in ./StoreLogMonitorComponent class StoreLogMonitorComponent - inline template:1:4 caused by : No provider for Dispatcher

**Comment by brandonroberts** [comment]
Are you using the latest release?

**Comment by k1ng440** [comment]
yes.

```
    "@ngrx/core": "^1.1.0",
    "@ngrx/effects": "^2.0.0-beta.3",
    "@ngrx/store": "^2.1.2",
    "@ngrx/store-devtools": "^3.0.1",
    "@ngrx/store-log-monitor": "^3.0.1",
```

**Comment by brandonroberts** [comment]
Can you provide some more information? Show your module where you are using the `logMonitor`

**Comment by MikeRyanDev** [comment]
Fixed in v3.0.2
